src/test/java/com/basho/riak/client/core/operations/itest/ITestBase.java

```diff
@@ -156,10 +176,21 @@ public static void setUp() throws CertificateException, IOException, KeyStoreExc
         cluster = new RiakCluster.Builder(builder.build()).build();
         cluster.start();
     }
-
+    
+    @Before
+    public void beforeTest() throws InterruptedException, ExecutionException
```

Comment:
I'd rather not do a full bucket reset / cleanup before and after every test for all test cases - it really slows down the test run.  
I did make a separate test base that does that if you need it though:   https://github.com/basho/riak-java-client/blob/3db4a7d313d6622614c788be62932a55e01354f1/src/test/java/com/basho/riak/client/core/operations/itest/ITestAutoCleanupBase.java